vaadin-app-layout-flow-parent/vaadin-app-layout-flow-integration-tests/src/main/java/com/vaadin/flow/component/applayout/examples/I18nPage.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.applayout.examples;
+
+import com.vaadin.flow.component.applayout.AppLayout;
+import com.vaadin.flow.component.applayout.AppLayout.AppLayoutI18n;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.router.Route;
+
+@Route("vaadin-app-layout/i18n")
+public class I18nPage extends Div {
```

Comment:
One final thing: Would be good to align the names of the page / test classes, I would suggest to use `AppLayoutI18nPage` here

Reply:
Done.